### ISSUE TITLE
if my pytorch version is 1.1, where should i change in the code to get the same result?

### ISSUE THREAD

**Issue by 714333462** [gmayday1997/SceneChangeDet#13]
i trained in pytorch 1.1, and get the loss as showed below
![image](https://user-images.githubusercontent.com/41322311/65826171-d1988300-e2b2-11e9-9c76-39fbc2a339d2.png)
where can i change in the code to get the right result?

**Comment by 714333462** [comment]
oh, my torch version is 0.4.1, so where can i change?

**Comment by neverstoplearn** [comment]
> oh, my torch version is 0.4.1, so where can i change?

have you solve you problem? I meet the same error  how did you fix it? thanks.

**Comment by LucasMesdon** [comment]
I meet the same error, did you fix it? And how ?
